ArduPlane/RC_Channel.cpp

```diff
@@ -116,6 +116,20 @@ void RC_Channel_Plane::do_aux_function_soaring_3pos(AuxSwitchPos ch_flag)
     plane.g2.soaring_controller.set_pilot_desired_state(desired_state);
 #endif
 }
+
+void RC_Channel_Plane::do_aux_function_flare(AuxSwitchPos ch_flag)
+{
+        switch(ch_flag) {
+        case AuxSwitchPos::HIGH:
+            plane.flare_switch_active = true;
+            break;
+        case AuxSwitchPos::MIDDLE:
```

Comment:
MIDDLE is normally do nothing for 2 pos switch (add a break)

Reply:
done